src/components/content/order/formElement/Common.tsx

```diff
@@ -0,0 +1,98 @@
+/*

```

Comment:
if the file does not contain any React component, we can keep the file extension to only ".ts"